java/src/main/native/src/ColumnViewJni.cu

```diff
@@ -15,8 +15,10 @@
  */
 
 #include <cudf/column/column_device_view.cuh>
+#include <cudf/column/column_factories.hpp>
 #include <cudf/detail/iterator.cuh>
 #include <cudf/detail/valid_if.cuh>
+#include <cudf/strings/detail/utilities.cuh>
```

Comment:
If you're having to include a header from a `detail` folder, something is wrong. These can break at any time, for any reason, and there will be no sympathy for any code that breaks.